mkosi/distributions/debian.py

```diff
@@ -292,12 +292,12 @@ def invoke_apt(
         operation,
         *extra,
     ]
-    env = dict(
-        APT_CONFIG=config_file,
-        DEBIAN_FRONTEND="noninteractive",
-        DEBCONF_INTERACTIVE_SEEN="true",
-        INITRD="No",
-    )
+    env: dict[str, PathString] = {
+        "APT_CONFIG": config_file,
+        "DEBIAN_FRONTEND": "noninteractive",
+        "DEBCONF_INTERACTIVE_SEEN": "true",
+        "INITRD": "No",
+    }
```

Comment:
Why chance this? I prefer using `dict()` over `{}`

Reply:
I can leave the `dict` call if you want, but generally using the literal for initialisation is preferable, since it allows for the little compile time optimisation that Python does. Using the literal it can become a dict (or e.g. in the case of sets a set or a frozen set) at compile time, whereas the dict call will only happen at runtime (which also means that there is a call happening). This is because Python will have to resolve the constructor, since it might have been overridden.

Reply:
There's a lot more pressing stuff to fix if we care about performance, let's leave the dict() as it is

Reply:
Sure, no problem. I do find literals much easier on the eyes, than functions calls, but let's leave that for another day. :)